### PR TITLE
add HandyWinGet-GUI

### PR DESCRIPTION
**Pull request by ghost1372**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1862)

### CODE REVIEW

**Comment by cstayyab** [comment]
Error Occurred while installing using manifest file

`winget install -m handyget.yaml`

```
Found HandyWinget-GUI [HandyOrg.HandyWinget-GUI]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/HandyOrg/HandyWinGet-GUI/releases/download/v1.3.0.1/HandyWinGet_x86.msix
  ██████████████████████████████  63.1 MB / 63.1 MB
Successfully verified installer hash
Starting package install...
Install failed: error 0x800B0109: The root certificate of the signature in the app package or bundle must be trusted.
0x80070570 : The file or directory is corrupted and unreadable.
```

**Comment by ghost1372** [comment]
thank you @cstayyab 
so there is a issue with signature
I'm trying to change the signature

File: manifests/HandyOrg/HandyWinGet-GUI/1.3.0.1.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+License: MIT
+LicenseUrl: https://github.com/HandyOrg/HandyWinGet-GUI/blob/master/LICENSE
+AppMoniker: handywinget
+Tags: winget, handy, handycontrol, handywinget, gui, package, packages, package manager 
+Description: GUI for installing apps through WinGet and Creating Yaml file
+Homepage: https://github.com/HandyOrg/HandyWinGet-GUI
+InstallerType: MSIX
+Installers:
+  - Arch: x86
+    Url: https://github.com/HandyOrg/HandyWinGet-GUI/releases/download/v1.3.0.1/HandyWinGet_x86.msix
```
Comment (cstayyab) [inline manifests/HandyOrg/HandyWinGet-GUI/1.3.0.1.yaml:14]: The new URL should be:

https://github.com/HandyOrg/HandyWinGet-GUI/releases/download/v1.3.0.1/setup.exe

Also change the Installer type to EXE

Comment (ghost1372) [reply manifests/HandyOrg/HandyWinGet-GUI/1.3.0.1.yaml:14]: This setup also has a signature and smart screen problem I'm looking for a way to sign setup without any problems

**Pull request closed without merging**